.circleci/config.yml

```diff
@@ -918,7 +918,9 @@ jobs:
           python: "$EMSDK_PYTHON"
       - run-tests:
           title: "crossplatform tests"
-          test_targets: "--crossplatform-only"
+          # Skipping test_strftime_zZ_gb_locale since setting LC_ALL causes
+          # macOS python to fail.
```

Comment:
Is this a MacOS python bug?

And is this failure in emcc code (which users run) or test harness code (which they do not)?

Reply:
IIUC if you run with a different locale such as `en_GB` then python expects that OS to have that locale installed and configured.

Its a failure of python, which in turn is a failure of emcc, to be runnable when `LC_ALL` is set to a missing/non-existent locale. 

I don't fully understand how locales get installed on macOS but this seems like a reasonable failure

Reply:
Thanks. And this only affects that test, I guess, because it forces that locale.

Reply:
```suggestion
          # macOS python to fail (the test forces a locale not installed there).
```